blogs/2021/01/ext-bisect.md

```diff
@@ -0,0 +1,40 @@
+# “Just like git-bisect, but for VS Code extensions.”
+
+The true power of VS Code is its extensions: theme extensions add colors and icons, language extensions provide IntelliSense and enable code navigation, debugger extensions enable you to drill into bugs. There are extensions that play music, some that show stock tickers, and there are extensions that enable collaborative work across locations and time zones. The VS Code marketplace hosts more than 23K extensions and it is not uncommon that users have 50+ extensions installed. With so many extensions bugs are inevitable and instead of denial we want to make trouble shooting easy.
```

Comment:
Should you like to the interesting extensions that "play music", "show stock tickers", and "enable collaboration"?

Reply:
I don't really wanna pick a real extension unless they are a clear winner. LiveShare might be one of them